<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launcher</class>
 <widget class="QMainWindow" name="launcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>launcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextBrowser" name="releaseNotesText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent; border: 1px solid black;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.TimesNewRoman'; font-size:18pt;&quot;&gt;Release Notes&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.TimesNewRoman'; font-size:12pt;&quot;&gt;Coming Soon&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>2</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/750x500_bg1.png);
 background-repeat: no-repeat;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="installprogressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>362</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="releasesComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>V0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hostServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Host a Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_runSettings">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Open Run Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generateSeed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Generate a Seed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ipAddress">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 5px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Server IP: 127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 5px;

color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Username: player1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Settings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;
qproperty-icon: url(:/resources/cog-svgrepo-com.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Connect with User/IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startAll">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Start All (Solo/Offline)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rememberMecheckBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Remember Me</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/minus.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/close.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="downloadLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Downloading: </string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>releaseNotesText</zorder>
   <zorder>installprogressBar</zorder>
   <zorder>releasesComboBox</zorder>
   <zorder>pushButton_hostServer</zorder>
   <zorder>pushButton_runSettings</zorder>
   <zorder>pushButton_generateSeed</zorder>
   <zorder>lineEdit_ipAddress</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>pushButton_Settings</zorder>
   <zorder>pushButton_connect</zorder>
   <zorder>pushButton_startAll</zorder>
   <zorder>rememberMecheckBox</zorder>
   <zorder>minusButton</zorder>
   <zorder>closeButton</zorder>
   <zorder>downloadLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
